src/v/http/client.cc

```diff
@@ -195,6 +202,10 @@ ss::future<iobuf> client::response_stream::recv_some() {
                 ec);
           }
           return ss::make_ready_future<iobuf>(std::move(out));
+      })
+      .handle_exception_type([this](const ss::tls::verification_error& err) {
+          return _client->shutdown().then(
```

Comment:
same here. should we log some context here in logger.error level.

Reply:
same here, it's not very useful to selectively log errors in the subsystem, IMO

Reply:
got it.